### PR TITLE
Added default value of Full for DatePicker.FlyoutPlacement and a sample for DatePicker

### PR DESCRIPTION
**Pull request by francoisalbert**
GitHub Issue (If applicable): #

## PR Type
What kind of change does this PR introduce?

 - Bugfix 

 - Documentation content changes 

## What is the current behavior?
DatePicker flyout placement not at bottom by default on iOS

## What is the new behavior?
Set the flyout placement to 'Full' by default so it appears at bottom

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

If no flyout placement was set, DatePicker flyout will now appear at the bottom instead of center

## Other information

Added sample for DatePicker too.

Internal Issue (If applicable):
https://nventive.visualstudio.com/Umbrella/_workitems/edit/150978/

### CODE REVIEW

File: src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/DatePicker/DatePickerSample.xaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+	<controls:SampleControl SampleDescription="Sample for Date Picker">
+		<controls:SampleControl.SampleContent>
+			<DataTemplate>
+				<Grid Background="White">
+					<DatePicker Grid.Row="1"
+								Margin="15,10"
+								HorizontalAlignment="Center"
+								VerticalAlignment="Top"
+								FlyoutPlacement="Full">
```
Comment (MatFillion) [inline src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/DatePicker/DatePickerSample.xaml]: sample doesn't have to specify FlyoutPlacement to work properly if I understand correctly? it's in the default style?
I would remove it from here

File: src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Models/DatePickerViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+using Uno.UI.Samples.UITests.Helpers;
+using Windows.UI.Core;
+using Windows.UI.Xaml.Data;
+
+namespace SamplesApp.Windows_UI_Xaml_Controls.Models
+{
+	[Bindable]
+	public class DatePickerViewModel : ViewModelBase
+	{
+		private DateTime _Date = DateTime.Now.Date;
```
Comment (MatFillion) [inline src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Models/DatePickerViewModel.cs]:
```suggestion
		private DateTime _date = DateTime.Now.Date;
```
to follow guidelines

File: src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePicker.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Summary
+
+DatePicker is use to select a specific date.
+
+*Button showing time open the time picker popup. 
+Bind to the Date property of the control to set initial time.
+*By default minute increment is set to 1
+*Cancel button cancel the new time selection. You can also click outside the time picker to do the same.
+*Done/OK button save the new selected time.
```
Comment (MatFillion) [inline src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePicker.md]: I think 2 lines don't apply to datepicker
-MinuteIncrement is N/A
- DatePicker doesn't have Cancel button in its default style (it should but doesn't yet)

Also should replace "time" with "date" :)

File: src/Uno.UI/UI/Xaml/Style/Generic/DatePickerSelector.xaml [hunk 4]
```diff
@@ -23,6 +23,8 @@
 				</ControlTemplate>
 			</Setter.Value>
 		</ios:Setter>
+        <ios:Setter Property="FlyoutPlacement"
```
Comment (MatFillion) [inline src/Uno.UI/UI/Xaml/Style/Generic/DatePickerSelector.xaml]: I think this is the only necessary change. DatePicker.cs and DatePicker.iOS.cs don't need to change.

Comment (francoisalbert) [reply src/Uno.UI/UI/Xaml/Style/Generic/DatePickerSelector.xaml]: Not necessary but I did it to standardize with timepicker

File: src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Models/DatePickerViewModel.cs (hunk 2 above)
Comment (francoisalbert) [reply src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Models/DatePickerViewModel.cs]: Done

File: doc/ReleaseNotes/_ReleaseNotes.md [hunk 5]
```diff
@@ -3,6 +3,7 @@
 ## Next version
 
 ### Features
+* Added sample for DatePicker
```
Comment (jeromelaban) [inline doc/ReleaseNotes/_ReleaseNotes.md]: This is not a feature, you do not need to add it there.

**Pull request merged**